src/olympia/accounts/tests/test_views.py

```diff
@@ -511,6 +511,32 @@ def test_logged_in_disallows_login(self):
             format='json')
         assert not self.find_user.called
 
+    @mock.patch.object(views, 'generate_api_token', lambda u: 'fake-api-token')
+    def test_already_logged_in_add_api_token_cookie_if_missing(self):
```

Comment:
I think it would be good to add a separate test asserting that the cookie is *not reset* if it already exists